### PR TITLE
Add active links to footer

### PR DESCRIPTION
**Pull request by amitch23**
## Description

This PR enables the footer links and adds pages with accessibility and privacy policy statements. The following links are now active and open to new windows:

*  twitter icon goes to  @US_TRANSCOM
* facebook icon goes to https://www.facebook.com/USTRANSCOM/
* help me and email go to email link: transcom.scott.tcj5j4.mbx.ppcf@mail.mil
* phone number is (302) 469-6683 ((302) 4MY-MOVE)
* Accessibility link goes to internal page statement
* Privacy and Policy statement link goes to internal page statement
 
## Reviewer Notes

* The email address is not the friendliest to display, so I put "email us". Should we do this, or something else?

## Code Review Verification Steps

* [ ] All tests pass.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [ ] Any migrations/schema changes also update the diagram in docs/schema/dp3.sqs
* [ ] There are no aXe warnings for UI.
* [ ] This works in IE.
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @willowbl00
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Request review from a member of a different team.
* [ ] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/157747459) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots
screen shot 2018-05-28 at 11 32 07 pm

### CODE REVIEW

File: src/shared/Footer/index.jsx [hunk 1]
```diff
@@ -18,11 +18,24 @@ function Footer() {
...
-                <a className="usa-footer-primary-link">Help Me</a>
+                <a
+                  className="usa-footer-primary-link"
+                  href="mailto:transcom.scott.tcj5j4.mbx.ppcf@mail.mil"
+                >
+                  Help Me
+                </a>
+              </li>
+              <li className="usa-width-one-fourth usa-footer-primary-content">
+                <a className="usa-footer-primary-link" href="/accessibility">
```
Comment (tinyels) [inline src/shared/Footer/index.jsx]: If you use `react-router` `Link` component here, the page won't reload logged_in users a second time.

File: src/shared/Footer/index.jsx [hunk 2]
```diff
@@ -18,11 +18,24 @@ function Footer() {
...
+                  Accessibility
+                </a>
               </li>
               <li className="usa-width-one-fourth usa-footer-primary-content">
-                <a className="usa-footer-primary-link">
-                  Site policies (example)
+                <a
```
Comment (tinyels) [inline src/shared/Footer/index.jsx]: If you use `react-router` `Link` component here, the page won't reload logged_in users a second time.

File: src/shared/Footer/index.jsx (hunk 1 above)
Comment (amitch23) [reply src/shared/Footer/index.jsx]: added for both primary links

File: src/shared/Footer/index.jsx (hunk 2 above)
Comment (amitch23) [reply src/shared/Footer/index.jsx]: added

**Pull request merged**